librz/core/cmd_api.c

```diff
@@ -330,6 +340,25 @@ RZ_API RzCmdDesc *rz_cmd_get_desc(RzCmd *cmd, const char *cmd_identifier) {
 	return res;
 }
 
+/**
+ * \brief Set the default mode of the command descriptor, if the type allows it.
+ *
+ * Command descriptors that support multiple output modes can also have a
+ * default one. This function can be used to set it.
+ */
+RZ_API void rz_cmd_desc_set_default_mode(RzCmdDesc *cd, RzOutputMode mode) {
+	switch(cd->type) {
+	case RZ_CMD_DESC_TYPE_ARGV_MODES:
+		cd->d.argv_modes_data.default_mode = mode;
+		break;
+	case RZ_CMD_DESC_TYPE_ARGV_STATE:
+		cd->d.argv_state_data.default_mode = mode;
+		break;
+	default:
```

Comment:
Maybe `rz_warn_if_reached()`?

Reply:
No, I think it is fine without it, so the caller of this function does not have to know the type of the cd but it can always call the function without having errors.